tmt/steps/report/reportportal.py

```diff
@@ -38,113 +25,191 @@ class ReportReportPortalData(tmt.steps.report.ReportStepData):
         option="--token",
         metavar="TOKEN",
         default=os.environ.get("TMT_REPORT_REPORTPORTAL_TOKEN"),
-        help="The token to use for upload to the ReportPortal instance.")
-    project: Optional[str] = tmt.utils.field(
+        help="The token to use for upload to the ReportPortal instance (from RP user profile).")
+    project: Optional[str] = tmt.utils.field(  # todo: argument should be mandatory
         option="--project",
-        metavar="PROJECT",
+        metavar="PROJECT_NAME",
         default=None,
         help="The project name which is used to create the full URL.")
-    launch_name: Optional[str] = tmt.utils.field(
-        option="--launch-name",
-        metavar="NAME",
+    launch: Optional[str] = tmt.utils.field(
+        option="--launch",
+        metavar="LAUNCH_NAME",
         default=None,
-        help="The launch name (base name of run id used by default).")
+        help="The launch name (name of plan per launch is used by default).")
+    env_var: List[str] = tmt.utils.field(
+        option="--env-var",
+        metavar="TMT_VAR",
+        default_factory=list,
+        multiple=True,
+        normalize=tmt.utils.normalize_string_list,
+        help="List one of the environment variables that begin with \"TMT_\" to disobey \
+the rule of not displaying them in Report Portal launch (the argument can be repeated).")
```

Comment:
@psss @happz @lukaszachy @thrix 
Need more insights here..
I have included listing all environment variables (`discover.tests()` -> `environment`) in RP Item Details as parameters.
As "TMT_" env variables tend to be unique, they have been breaking the history aggregation, so I omitted them and added the option to include them if needed (`--env-var`).

Recently, I had noticed that this aggregation problem is solved when Test Case ID is stated (`discover.tests()` -> `id`).
See the [example](https://reportportal-rhel.apps.ocp-c1.prod.psi.redhat.com/ui/#test_tmt/launches/latest/2243): `/test_01` (with Test Case ID) vs `test_02` (without Test Case ID)
![image](https://github.com/teemtee/tmt/assets/80918188/9394789c-8ea4-4f45-9b49-be67c1c503bd)
![image](https://github.com/teemtee/tmt/assets/80918188/60f12d0a-696e-4a7c-b721-fbc37a0b7404)

Also I realize that `--env-var` can seem quite confusing if all the rest environment variables are already reported by default. (See [the conversation above](https://github.com/teemtee/tmt/pull/2088#discussion_r1199105937).)
What are your opinions here about the need of `--env-var` option? Should I change the approach? Or its functionality? Or any other ideas?

Reply:
@happz Is there any additional parameter so `is_flag=True` works also for fmf data?